wrapper/src/main/java/software/amazon/jdbc/util/RdsUtils.java

```diff
@@ -166,6 +166,21 @@ public class RdsUtils {
           ".*(?<prefix>-green-[0-9a-z]{6})\\..*",
           Pattern.CASE_INSENSITIVE);
 
+  private static final Pattern BG_GREEN_HOSTID_PATTERN =
+      Pattern.compile(
+          "(.*)-green-[0-9a-z]{6}",
+          Pattern.CASE_INSENSITIVE);
+
+  private static final Pattern BG_OLD_HOST_PATTERN =
+      Pattern.compile(
+          ".*(?<prefix>-old1)\\..*",
+          Pattern.CASE_INSENSITIVE);
+
+  private static final Pattern BG_PREFIX_HOST_PATTERN =
```

Comment:
If I understand correctly this is a pattern for a full host URL rather than just the prefix
```suggestion
  private static final Pattern BG_HOST_PATTERN =
```